transition-guide.md

```diff
@@ -18,29 +21,29 @@ The following transition guide is organized into a logical sequence of steps as
     * Step 5 - Testing and optimizing your workload
     * Step 6 - Performance testing
 * [Infrastructure and deployment](#infrastructure-and-deployment)
-    * Step 7 - Update your infrastructure as code
```

Comment:
I think this line should be left as-is. It's referring to Terraform, Pulumi, etc.

Reply:
sure, it has been reverted to initial statement.